### PR TITLE
add kiss hook for total build time

### PR DESCRIPTION
**Pull request by sewnie**
shamelessly stolen from ioraff's hk hook

### CODE REVIEW

File: kiss-buildtime-hook [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+# https://kisslinux.org/wiki/package-manager#006
+
+case $1 in
+    pre-build)
+        IFS=. read -r _start _ < /proc/uptime
+        printf '%d' "$_start" > "/tmp/$2_start"
```
Comment (illiliti) [inline kiss-buildtime-hook]: Always use `${TMPDIR:-/tmp}` instead of simple `/tmp`

File: kiss-buildtime-hook [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        rm "/tmp/$2_start"
+
+        _s=$((_end - _start))
+        _h=$((_s / 60 / 60 % 24))
+        _m=$((_s / 60 % 60 ))
+
+        [ "$_h" = 0 ] || _u="${_u}${_h}h "
+        [ "$_m" = 0 ] || _u="${_u}${_m}m "
+
+        printf '\033[1;33m-> \033[1;34m%s \033[m%s\n' "$2" "Build finished in ~${_u:-${_s}s}"
```
Comment (illiliti) [inline kiss-buildtime-hook]: Hm, i think it should print to stderr. Append `>&2`

**Review by illiliti — CHANGES_REQUESTED** [review]

File: kiss-buildtime-hook (hunk 2 above)
Comment (sewnie) [reply kiss-buildtime-hook]: why? it's not an error

Comment (illiliti) [reply kiss-buildtime-hook]: stderr is not limited to errors. All diagnostic messages should also go to stderr

Comment (sewnie) [reply kiss-buildtime-hook]: oh alright!

**Comment by illiliti** [comment]
Merged, thanks!

**Pull request merged**